spec/index.html

```diff
@@ -285,26 +285,26 @@ <h2>Notation and Terminology</h2>
     <p>Two graphs are <a data-cite="RDF12-CONCEPTS#dfn-graph-isomorphism">isomorphic</a> when each maps into the other by a 1:1 mapping on blank nodes. Isomorphic graphs are mutual instances with an invertible instance
       mapping. As blank nodes have no particular identity beyond their location in a graph, we will often treat isomorphic graphs as identical.</p>
 
-    <p >An RDF graph is <dfn>lean</dfn> if it has no instance which is
+    <p>An RDF graph is <dfn>lean</dfn> if it has no instance which is
       a <a>proper subgraph</a> of itself. Non-lean graphs have internal redundancy
-      and express the same content as their lean <a>subgraphs</a>. For example, the graph</p>
+      and express the same content as their lean <a>subgraphs</a>. For example, the graph in Example 1 is not lean:</p>
 
     <pre class="example" title="Non-lean graph">
       ex:a ex:p _:x .
       _:y ex:p _:x .
     </pre>
 
-    <p >is not lean, but</p>
+    <p>In contrast, the graph in Example 2 <em>is</em> lean:</p>
 
     <pre class="example" title="Lean graph">
       ex:a ex:p _:x .
       _:x ex:p _:x .
     </pre>
 
-    <p>is lean. A <a>ground RDF graph</a> is lean. </p>
+    <p>A <a>ground RDF graph</a> is also lean.</p>
```

Comment:
The also is unneeded and only interferes with the meaning of the sentence.

Reply:
This whole paragraph does rather dangle. It would be better somewhere else, maybe after the sentence:

"An RDF graph is lean if it has no instance which is a [proper subgraph](https://www.w3.org/TR/rdf12-semantics/#dfn-proper-subgraph) of itself."